app/api/business/team_business.py

```diff
@@ -91,19 +91,23 @@ def get_people_overview():
     return people_overview
 
 
-def get_team(team_id):
+def get_team(team_id, allow_anyone=None):
     team = team_service.get_team(team_id)
     if not team:
         raise NotFoundError('Team {} does not exist'.format(team_id))
 
-    team_member = team_member_service.find(
-        is_team_lead=True,
-        team_id=team_id,
-        user_id=current_user.id
-    ).one_or_none()
+    if allow_anyone is None:
+        allow_anyone = False
 
-    if not team_member:
-        raise UnauthorisedError('Only team leads can edit a team')
+    if allow_anyone is False:
```

Comment:
Do you need to set `allow_anyone` to `False` (above)?  Is the same achieved if you check for `None` here?
```suggestion
    if allow_anyone is None:
```